benchmarks/source-cosmicjs/update-post.js

```diff
@@ -0,0 +1,34 @@
+require("dotenv").config({
+  path: `.env.${process.env.NODE_ENV || 'development'}`,
+})
+
+const Cosmic = require('cosmicjs')
+
+const bucket = Cosmic().bucket({
+  slug: process.env.BENCHMARK_COSMIC_BUCKET,
+  read_key: process.env.BENCHMARK_COSMIC_READ_KEY,
+  write_key: process.env.BENCHMARK_COSMIC_WRITE_KEY
+})
+
+const DATASET = (Number(process.env.BENCHMARK_COSMIC_DATASET) || 512)
+
+const randomDoc = Math.floor(Math.random() * DATASET)
+
+console.log(`Getting article number ${randomDoc}`)
+
+const run = async () => {
+  const { objects } = await bucket.getObjects({
+    skip: randomDoc,
+    limit: 1,
+    props: 'slug,title'
+  })
+  if (objects && objects.length > 0) {
+    const { slug, title } = objects[0];
+    await bucket.editObject({
+      slug,
+      title: `${title}!`
+    })
+    console.log(`Updated article number ${randomDoc}`)
```

Comment:
Since Cosmic doesn't send post update hooks, we can add them here like we do with md/mdx. https://github.com/duffn/gatsby/blob/3a1b7477afea16ca7e2407c774c1b26e466a6a31/benchmarks/md/scripts/data-update.ts#L38